tests/translator/input/function_with_deployment_preference_all_parameters.yaml

```diff
@@ -38,6 +39,7 @@ Resources:
       CodeUri: s3://my-bucket/myValidationTestFunction.zip
       DeploymentPreference:
         Enabled: false
+        Type: Linear10PercentEvery1Minute
```

Comment:
Why are you changing these? We should be first of all testing against the status quo, with no changes.

Reply:
It's a required field, but missing from the template. This results json schema check failure. 
https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-function-deploymentpreference.html#sam-function-deploymentpreference-type

Reply:
Let's skip it. Let's not change any templates currently. There is no formal schema at the moment. First step is to document what our tests consider as valid, as that is the effective source of truth (since that's what deal with the logic). We should document the discrepancies and later figure out whether docs need to change, or tests need to change.